Tasks/HelmDeployV0/src/helm.ts

```diff
@@ -66,15 +66,14 @@ function runHelmSaveCommand(helmCli: helmcli, kubectlCli: kubernetescli, failOnS
         throw new Error(tl.loc("SaveSupportedInHelmsV3Only"));
     }
     process.env.HELM_EXPERIMENTAL_OCI="1";
-    runHelm(helmCli, "saveChart", kubectlCli, failOnStderr);
+    //helm chart save has been replaced with helm package command in v3.7
```

Comment:
Is seems this change will be breaking for older versions of helm.  It would be better to support both code paths.

Also, if we change default behavior, we should use FF to enable

Reply:
Agree that we should support commands from both versions, will update here.
Looks that no functional or behavior changes to the PR, and there's one related test that needs to be updated accordingly.

Reply:
To-do list from offline sync up:)
- Add and enable FF since there's functional changes to the task when customers are using helm 3.7.0+
- Add more telemetry to it so that we can track their usages